website/content/docs/commands/alloc/checks.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: 'Commands: alloc checks'
+description: |
+  Outputs service health check status information.
+---
+
+# Command: alloc checks
+
+**Alias: `nomad checks`**
+
+The `alloc checks` command outputs the latest health check status information for checks in the allocation.
+
+## Usage
+
+```plaintext
+nomad alloc checks [options] <allocation>
+```
+
+Outputs the latest health check status information for services in the allocation
+using the Nomad service discovery provider. This command accepts an allocation
+ID or prefix as the sole argument.
+
+When ACLs are enabled, this command requires a token with the read-job and list-jobs 
+capabilities for the allocation's namespace.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Exec Options
+
+- `-verbose`: Display verbose output.
+
+- `-json`: Output the allocation in its JSON format.
+
+## Examples
+
+Short checks of an alloc health check using its short id:
+
+```shell-session
+$ nomad alloc checks e0fdbd85
+
+Status of 1 Nomad Service Checks
+
+ID         =  9f4e18fd0867cebb19a8fac3d7a1cf27
+Name       =  alive
+Group      =  example.cache[0]
+Task       =  (group)
+Service    =  redis-cache
+Status     =  success
+Mode       =  healthiness
+Timestamp  =  2023-03-09T16:10:23+01:00
+Output     =  nomad: tcp ok
+```
+
```

Comment:
Lets also add an example each for using the JSON and template flag, so new users have the easiest time learning and understanding what is possible.

Reply:
Good idea